web-client/src/views/PendingReport/PendingReportList.jsx

```diff
@@ -24,12 +24,18 @@ export const PendingReportList = connect(
         <div className="grid-row margin-bottom-2">
           <div className="tablet:grid-col-8">
             <div className="grid-row grid-gap">
-              <h3 id="filterHeading">Show items for</h3>
+              <label
+                className="dropdown-label-serif margin-right-3"
+                htmlFor="inline-select"
+                id="pending-reports-filter-label"
+              >
+                Show items for
+              </label>
               <BindedSelect
-                aria-describedby="case-deadlines-tab filterHeading"
-                aria-label="judge"
+                aria-describedby="pending-reports-filter-label"
+                aria-label="judge filter"
                 bind="screenMetadata.pendingItemsFilters.judge"
-                className="select-left"
+                className="select-left inline-select width-mobile"
```

Comment:
What does width-mobile do?

Reply:
good q - it's from USWDS https://designsystem.digital.gov/utilities/height-and-width/

i was trying to use a class that makes this select a more similar width to what it was before my changes. the name of it is confusing but it's 320px wide!

Reply:
Got it, thank you!